internal/pkg/manifest/workload.go

```diff
@@ -72,6 +72,7 @@ type Workload struct {
 type Image struct {
 	Build        BuildArgsOrString `yaml:"build"`           // Build an image from a Dockerfile.
 	Location     *string           `yaml:"location"`        // Use an existing image instead.
+	Credential   *string           `yaml:"credential"`      // ARN of the secret containing the private repository credentials.
```

Comment:
Can we rename this to:
```suggestion
	Credentials   *string           `yaml:"credentials"`      // ARN of the secret containing the private repository credentials.
```
So that we are consistent with the task definition.

Reply:
Done.